src/legacy/core_plugins/interpreter/index.js

```diff
@@ -25,6 +25,9 @@ export default function (kibana) {
     id: 'interpreter',
     require: ['kibana', 'elasticsearch'],
     publicDir: resolve(__dirname, 'public'),
+    uiExports: {
+      injectDefaultVars: server => ({ defaultSpaceBasePath: server.config().get('server.basePath') }),
```

Comment:
We've tried to keep the "space" terminology out of OSS, so I'd like to do so here as well. How about `serverRootBasePath` or `serverBasePath`?